tests/Analyze/Eval.hs

```diff
@@ -0,0 +1,138 @@
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+module Analyze.Eval where
```

Comment:
i guess we keep this stuff in test-land for now, and move it when we need it on the library side?